### PR TITLE
Document new Portal parts

### PR DESCRIPTION
**Pull request by benoitgrelard**
https://github.com/radix-ui/primitives/pull/1425
https://github.com/radix-ui/primitives/pull/1426
https://github.com/radix-ui/primitives/pull/1427
https://github.com/radix-ui/primitives/pull/1429

### CODE REVIEW

File: data/primitives/components/context-menu/0.1.7.mdx [hunk 1]
```diff
@@ -171,6 +175,33 @@ The area that opens the context menu. Wrap it around the target you want the con
...
+    {
+      name: 'forceMount',
+      type: 'boolean',
+      description: (
+        <span>
+          Used to force mounting when more control is needed. Useful when
+          controlling animation with React animation libraries. If used on this
+          part, it will be inherited by <Code>ContextMenu.Content</Code> and{' '}
+          <Code>ContextMenu.SubContent</Code> respectively.
+        </span>
```
Comment (andy-hook) [inline data/primitives/components/context-menu/0.1.7.mdx:193]: I like these expanded descriptions mentioning inheritance, perhaps we should align `Dialog` to include these as well?

File: data/primitives/components/tooltip/0.1.8.mdx [hunk 2]
```diff
@@ -203,6 +221,17 @@ The component that pops out when the tooltip is open.
...
+      name: 'forceMount',
+      type: 'boolean',
+      description: (
+        <span>
+          Used to force mounting when more control is needed. Useful when
+          controlling animation with React animation libraries. It inherits from{' '}
+          <Code>Tooltip.Portal</Code>.
+        </span>
+      ),
+    },
```
Comment (andy-hook) [inline data/primitives/components/tooltip/0.1.8.mdx:234]: 👍

File: data/primitives/overview/releases.mdx [hunk 3]
```diff
@@ -25,6 +25,7 @@ metaDescription: Radix Primitives releases and their changelogs.
 - Add `data-highlighted` attribute to support styling <PRLink id={1388} />
 - [**Breaking**] Improve indirect nesting of context menus. Submenus must now be created using explicit parts. <PRLink id={1394} />
 - [**Breaking**] Move `allowPinchZoom` to root. <PRLink id={1400} />
+- [**Breaking**] Add new `Portal` part. To avoid regressions, use this part if you want portalling behavior. Note that `z-index` isn't managed anymore so you have full control of layering. <PRLink id={1429} />
```
Comment (andy-hook) [inline data/primitives/overview/releases.mdx:28]: I don't think we've been super consistent about it in the past but just checking, is American english our preference?

**Review by andy-hook — COMMENTED** [review]
Looks good, only notable comment is regarding consistency in the `Dialog` docs.

We should also remove the reference to `portalled` in the home page code slider:

https://github.com/radix-ui/website/blob/5083be0b1d6318061cab86da484d679765782cb9/components/marketing/DeveloperExperienceSection.tsx#L436

File: data/primitives/overview/releases.mdx (hunk 3 above)
Comment (benoitgrelard) [reply data/primitives/overview/releases.mdx:28]: I would say yes, given our global audience.

File: data/primitives/components/context-menu/0.1.7.mdx (hunk 1 above)
Comment (benoitgrelard) [reply data/primitives/components/context-menu/0.1.7.mdx:193]: I'm confused, it should already be the case no? That's where I got it from.

Comment (andy-hook) [reply data/primitives/components/context-menu/0.1.7.mdx:193]: Whoops, I was looking at the live url 🙈

**Review by andy-hook — APPROVED** [review]

**Pull request merged**